api/sys_mounts.go

```diff
@@ -130,6 +130,7 @@ type MountConfigInput struct {
 	DefaultLeaseTTL string `json:"default_lease_ttl" structs:"default_lease_ttl" mapstructure:"default_lease_ttl"`
 	MaxLeaseTTL     string `json:"max_lease_ttl" structs:"max_lease_ttl" mapstructure:"max_lease_ttl"`
 	ForceNoCache    bool   `json:"force_no_cache" structs:"force_no_cache" mapstructure:"force_no_cache"`
+	PluginName      string `json:"plugin_name" structs:"plugin_name" mapstructure:"plugin_name"`
```

Comment:
@briankassouf should we `omitempty` plugin_name in here (and elsewehere where it appears)? Right now some tests are failing because it doesn't expect `plugin_name` as part of the response body for mount-related requests.